api.go

```diff
@@ -32,21 +32,17 @@ var (
 	// ErrEnqueueTimeout is returned when a command fails due to a timeout.
 	ErrEnqueueTimeout = errors.New("timed out enqueuing operation")
 
-	// ErrKnownPeer is returned when trying to add a peer to the configuration
```

Comment:
Only difference now is that if you call the peer change functions in a way that should be a no-op it'll make the changes anyway and run them through the Raft log. Consul would rely on this since it called these functions over and over again to reconcile things.